### ISSUE TITLE
[BUG] Security and setup check fails in 28.0.1

### ISSUE THREAD

**Issue by ntompson** [linuxserver/docker-nextcloud#409]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When navigating to the Administration Overview page(`/settings/admin/overview`), the automated checks fail:

<img width="833" alt="Screenshot 2024-01-10 at 10 34 09 am" src="https://github.com/linuxserver/docker-nextcloud/assets/32885870/0cb0b5c4-6caa-444c-8121-981a8d635a85">

The java console reports 3 errors, which I have not seen before:

<img width="878" alt="Screenshot 2024-01-10 at 10 34 48 am" src="https://github.com/linuxserver/docker-nextcloud/assets/32885870/fb49a210-7bf4-4b3f-b81e-a3ad758579ca">

### Expected Behavior

In previous versions, the automated checks complete and either give me an all OK message, or point to issues that need to be resolved. I've never seen this failure before

### Steps To Reproduce

As above, in a browser, navigate to `/settings/admin/overview` and see error message after waiting for some time for automated checks to complete.

### Environment

```markdown
- OS: Debian bullseye (11.8) (Linux xxxxx 5.10.0-26-amd64 #1 SMP Debian 5.10.197-1 (2023-09-29) x86_64 GNU/Linux)
- How docker service was installed: distro package manager (apt)
```

### CPU architecture

x86-64

### Docker creation

YAML docker compose file:

```yaml
---
version: "3"
services:
  nextcloud:
    image: linuxserver/nextcloud:28.0.1
    container_name: nextcloud
    environment:
      - PUID=912
      - PGID=912
      - TZ=Australia/Hobart
    volumes:
      - /home/nick/docker/nextcloud/config:/config
      - /mnt/Calypso/nc-data:/data
    depends_on:
      - redis
    restart: unless-stopped
    networks:
      - swag_proxy-net
      - redis-net

  redis:
    image: redis:7.2.3
    volumes:
      - /home/nick/docker/redis:/data
    restart: unless-stopped
    container_name: redis
    networks:
      - redis-net

networks:
  swag_proxy-net:
    external: true
  redis-net:
    driver: bridge
```

### Container logs

```bash
docker logs nextcloud
[migrations] started
[migrations] 01-nginx-site-confs-default: skipped
[migrations] 02-default-location: skipped
[migrations] done
───────────────────────────────────────

      ██╗     ███████╗██╗ ██████╗ 
      ██║     ██╔════╝██║██╔═══██╗
      ██║     ███████╗██║██║   ██║
      ██║     ╚════██║██║██║   ██║
      ███████╗███████║██║╚██████╔╝
      ╚══════╝╚══════╝╚═╝ ╚═════╝ 

   Brought to you by linuxserver.io
───────────────────────────────────────

To support LSIO projects visit:
https://www.linuxserver.io/donate/

───────────────────────────────────────
GID/UID
───────────────────────────────────────

User UID:    912
User GID:    912
───────────────────────────────────────

using keys found in /config/keys
Initializing nextcloud 28.0.1.1 (this can take a while) ...
Setting permissions
Initializing finished
[custom-init] No custom files found, skipping...
[ls.io-init] done.
```

**Comment by ntompson** [comment]
I should add: I have updated the `default.conf` file to the latest version as directed. I am also running through the SWAG reverse proxy using all latest config.

**Comment by ntompson** [comment]
An update. The 504 issue has disappeared (and consequently the security and setup check completes correctly. However, the two 404s remain. I don't know whether they matter...?

**Comment by DoubleAps1** [comment]
> I should add: I have updated the `default.conf` file to the latest version as directed. I am also running through the SWAG reverse proxy using all latest config.

Thanks for this. I was able to fix the external scan and now it identifies the correct version I'm running. Unfortunately, the internal scan still fails with the error that you specified in the 1st post.

Just to provide path for anyone else. The file is located in the path below. I just ended up renaming the current `default.conf` and renaming `default.conf.sample` to `default.conf`.

`/config/nginx/site-confs`

**Comment by homerr** [comment]
Closing out as this was likely a result of bugs in NC 28 prior to 28.0.3 - no need to worry about the 404's they relate to the certificate generation (if needed).

**Comment by ntompson** [comment]
Why was this closed? It is still an issue in 28.0.4:

<img width="710" alt="Screenshot 2024-04-02 at 1 22 52 PM" src="https://github.com/linuxserver/docker-nextcloud/assets/32885870/2739fe54-df84-43e4-992e-d47eb291a1c6">

The setup checks take forever to run (they used to come back quickly). There's clearly something wrong here. My config is as recommended by you guys -- not sure what else I can do?
